x/twap/listeners_test.go

```diff
@@ -285,7 +300,13 @@ func (s *TestSuite) TestAfterSwap_JoinPool() {
 				s.RunBasicJoin(poolId)
 			}
 
-			// test that either of swapping in a pool or joining a pool
+			if tc.exitPool {
+				poolCreator := s.TestAccs[0]
+				lpBalance := s.App.BankKeeper.GetBalance(s.Ctx, s.TestAccs[0], "gamm/pool/1").Amount
+				s.App.GAMMKeeper.ExitPool(s.Ctx, poolCreator, poolId, lpBalance.Quo(sdk.NewInt(10)), sdk.Coins{})
+			}
+
+			// test that either of swapping in a pool, joining a pool, or exiting a pool
```

Comment:
Do you think we should make a `s.RunBasicExit` in the test suite that we can use for future tests?

Reply:
That would be great to do

Reply:
Done!